ui/src/shared/constants/colorOperations.ts

```diff
@@ -88,10 +85,6 @@ export const generateThresholdsListHexs = ({
       lastValueNumber
     )
 
-    if (cellType === 'table') {
```

Comment:
removing these lines reverted the functionality to its original format, restoring it to what it was originally supposed to do